srcpkgs/switchboard/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'switchboard'
+pkgname=switchboard
+version=2.3.4
+revision=1
+wrksrc="${pkgname}-${version}"
```

Comment:
No need, this is the default.

Reply:
heh, leftover from when I was packing a commit rather than a release